coremltools/test/neural_network/test_graph_passes.py

```diff
@@ -250,13 +796,6 @@ def _build_and_test_network(input_size, transpose_layers, expected_layers):
                     spec.layers[output_layer_idx].transpose.axes
                 )
 
-        _build_and_test_network(
```

Comment:
@sberardi-apple I removed these two test case since if the merged op belongs to outputs produces some issue when compiling the model.